gemviz23/demo/resultwindow.py

```diff
@@ -78,34 +79,40 @@ def headerData(self, section, orientation, role=QtCore.Qt.DisplayRole):
 
     def doPager(self, action, value = None):
         print(f"doPager {action =}, {value =}")
+
+        catalog_length = self.catalog_length()
+        offset = self.pageOffset()
+        size = self.pageSize()
+        print(f"{catalog_length=} {offset=}  {size=}")
```

Comment:
We can remove this print statement

Reply:
Taking out all similar prints in this module by comment